### PR TITLE
Fixes IE by changing document.contains to jQuery

### PR DESCRIPTION
**Pull request by jordansmith42**

### CODE REVIEW

File: src/mixins.coffee [hunk 1]
```diff
@@ -72,7 +72,7 @@ Ember.Widgets.BodyEventListener = Ember.Mixin.create
       Ember.run =>
         if (@get('_state') or @get('state')) is 'inDOM' and Ember.isEmpty(@$().has($(event.target)))
           # check if event.target still exists in DOM
-          if document.contains(event.target)
+          if $.contains(document.body, event.target)
```
Comment (Dylnuge) [inline src/mixins.coffee]: Can't it be `$('body').contains`?

**Comment by philpee2** [comment]
The title of this PR presumes that IE can be fixed, which is clearly wrong. Even Microsoft agrees. 

Also, tests?

**Comment by jordansmith42** [comment]
@philpee2 we will be implementing Open Sauce to catch these types of issues, also we'll look into just having a linter to catch the common cases.

**Comment by philpee2** [comment]
Sounds good. Although I still don't quite understand the cause of the bug. Does IE really not support `.contains`?

**Comment by Dylnuge** [comment]
No, this is Chrome's fault. IE is right in this case. `document.contains` isn't a thing. It's an alias for `document.body.contains`

**Comment by Dylnuge** [comment]
LGTM

**Comment by philpee2** [comment]
Oh I see. That makes sense.

**Comment by Dylnuge** [comment]
Actually `document.contains` should be a thing, as a document should be a fully valid node interface. Not sure why it doesn't work.

**Comment by cyril-sf** [comment]
@philpee2 We are going to use Saucelabs to do cross browser testing ( https://saucelabs.com )

**Comment by thangdinh** [comment]
@Dylnuge https://developer.mozilla.org/en-US/docs/Web/API/Node/contains

**Comment by Dylnuge** [comment]
Wait this needs dist files.

Also @thangdinh I saw that issue but it is in fact considered a node in IE: https://msdn.microsoft.com/en-us/library/ms535862(v=vs.85).aspx. It just seems that their `Node.contains` isn't implemented for documents, for some reason?

**Comment by thangdinh** [comment]
Right, we need dist files here.

**Comment by jordansmith42** [comment]
@thangdinh I'm working on the dist files, github is having issues which is causing issues with the build: https://status.github.com/

**Comment by jordansmith42** [comment]
@Dylnuge @thangdinh I added this dist files in this PR: https://github.com/Addepar/ember-widgets/pull/214

**Comment by Dylnuge** [comment]
It looks like IE contains is implemented on the element level.

**Pull request merged**